packages/experimental/Badge/src/PresenceBadge/PresenceBadge.tsx

```diff
@@ -35,22 +43,21 @@ export const PresenceBadge = compose<PresenceBadgeType>({
   ...stylingSettings,
   slots: {
     root: View,
-    svgXml: SvgXml,
+    svg: Svg,
   },
   useRender: (userProps: PresenceBadgeProps, useSlots: UseSlots<PresenceBadgeType>) => {
-    const badge = useBadge(userProps) as PresenceBadgeProps;
-    const Slots = useSlots(badge, (layer) => badgeLookup(layer, badge));
+    const Slots = useSlots(userProps, (layer) => prensenceBadgeLookup(layer, userProps));
 
     return (final: PresenceBadgeProps) => {
-      const { size, status, outOfOffice, ...mergedProps } = mergeProps(badge, final);
+      const { size, status, outOfOffice, ...mergedProps } = mergeProps(userProps, final);
       const isOutOfOffice = outOfOffice || false;
-      const iconXml = `<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-        ${getIconPath(status, isOutOfOffice)}
-      </svg>`;
+      const path = getIconPath(status, isOutOfOffice);
 
       return (
         <Slots.root {...mergedProps}>
-          <Slots.svgXml xml={iconXml} />
+          <Slots.svg viewBox="0 0 16 16" fill="none">
+            <Path fill="currentColor" d={path} />
+          </Slots.svg>
```

Comment:
Should these props move to slotProps? I know they're not styling so I suppose it makes to keep  them inline, I'm not sure.

Reply:
I think it would be good to use Icon component later. But we'll start working on it when Avatar is finished.